cfy_manager/components/composer/composer.py

```diff
@@ -45,8 +45,10 @@
 NODEJS_DIR = join('/opt', 'nodejs')
 LOG_DIR = join(BASE_LOG_DIR, COMPOSER)
 
-DB_CLIENT_KEY_PATH = join(CONF_DIR, 'db_client.key')
-DB_CLIENT_CERT_PATH = join(CONF_DIR, 'db_client.crt')
+# These are all the same key as the other db keys, but postgres is very strict
+# about permissions (no group or other permissions allowed)
+DB_CLIENT_KEY_PATH = join('/etc/cloudify/ssl/composer_db.key')
+DB_CLIENT_CERT_PATH = join('/etc/cloudify/ssl/composer_db.crt')
```

Comment:
join a single literal path? presumably you meant some constant?

Reply:
Good catch, changed to just make them constants.